en/news/_posts/2019-12-17-ruby-2-7-0-rc1-released.md

```diff
@@ -0,0 +1,363 @@
+---
+layout: news_post
+title: "Ruby 2.7.0-rc1 Released"
+author: "naruse"
+translator:
+date: 2019-12-17 12:00:00 +0000
+lang: en
+breaks: false
```

Comment:
What's the idea behind this tag?

Reply:
It's for HackMD which is used to edit draft but the tag will be removed in next version.